lib/components/map/default-map.tsx

```diff
@@ -269,7 +269,9 @@ class DefaultMap extends Component {
       this.context
     const { baseLayers, maxZoom, overlays } = mapConfig || {}
     const { lat, lon, zoom } = this.state
-    const vectorTilesEndpoint = makeApiUrl(config, 'vectorTiles', {})
+    const vectorTilesEndpoint = `${assembleBasePath(
+      config
+    )}/otp/routers/default/vectorTiles`
```

Comment:
Wow, `/otp/routers/default` should come from `config`.

Reply:
It is already the case in most of them.

Reply:
What do you mean? This route is going to not follow the `/routers/default` pattern very soon. Therefore the `path` config item should be deprecatable soon. We don't allow configuration of the graphql path, so why this one?

Reply:
But right now it does. It will also be easier to not miss this if referring to `config.api.path` once we switch to the new paths and get rid of `config.api.path`.

Reply:
What about a global constant that can be overridden in the config?

Reply:
```
    const vectorTilesEndpoint = `${assembleBasePath(
      config
    )}${config.api.path}/vectorTiles`
```

Reply:
We should discuss further. I want to deprecate `path`